auth/team_test.go

```diff
@@ -78,10 +78,12 @@ func (s *S) TestCreateTeamValidation(c *check.C) {
 		{"a", ErrInvalidTeamName},
 		{"@abc", ErrInvalidTeamName},
 		{"my team", ErrInvalidTeamName},
+		{"Abacaxi", ErrInvalidTeamName},
+		{"TEAM", ErrTeamAlreadyExists},
```

Comment:
using `validation.ValidateName` both `Abacaxi` and `TEAM` become invalid names, so I think the error to check should be `ErrInvalidTeamName`